src/lein_monolith/task/each.clj

```diff
@@ -37,6 +44,34 @@
       [:start start])))
 
 
+(defn- print-report
+  "Reports information about the tasks given a results map."
+  [results elapsed]
+  (let [task-time (reduce + (keep :elapsed results))
+        speedup (/ task-time elapsed)]
+    (lein/info (format "\n%s  %7.3f seconds"
+                       (ansi/sgr "Run time:" :bold :cyan)
+                       (/ elapsed 1000)))
+    (lein/info (format "%s %7.3f seconds"
+                       (ansi/sgr "Task time:" :bold :cyan)
+                       (/ task-time 1000)))
+    (lein/info (format "%s   %7.1f"
+                       (ansi/sgr "Speedup:" :bold :cyan)
+                       speedup))
+    (lein/info (->> results
+                    (sort-by :elapsed)
+                    (reverse)
+                    (take 8)
+                    (map #(format "%-40s %s %.3f seconds"
+                                  (ansi/sgr (:name %) :bold :yellow)
+                                  (if (:success %) " " "!")
+                                  (/ (:elapsed %) 1000)))
+                    (str/join "\n")
+                    (str \newline
+                         (ansi/sgr "Slowest projects:" :bold :cyan)
+                         \newline)))))
+
+
 (defn- select-projects
   "Returns a vector of pairs of index numbers and symbols naming the selected
   subprojects."
```

Comment:
Not necessary, but it is useful for ensuring the futures get constructed in a correct dependency order, so it's simpler to leave it in.